src/main/java/ladder/model/LadderGameResult.java

```diff
@@ -0,0 +1,51 @@
+package ladder.model;
+
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LadderGameResult {
+
+    private static final int ZERO = 0;
+    private final Map<Member, LadderGameReword> result;
+
+    private LadderGameResult(final Map<Member, LadderGameReword> result) {
+        this.result = Collections.unmodifiableMap(result);
+    }
+
+    public static LadderGameResult create(final Members members, final LadderGameRewords ladderGameRewords) {
+        Map<Member, LadderGameReword> ladderGameResult = IntStream.range(ZERO, members.count())
+            .boxed()
+            .collect(Collectors.toMap(members::get, ladderGameRewords::get, (item1, item2) -> item1, LinkedHashMap::new));
+
+        return new LadderGameResult(ladderGameResult);
+    }
+
+    public LadderGameReword get(final Member member) {
+        return result.get(member);
+    }
+
+    public Map<Member, LadderGameReword> getResult() {
+        return result;
```

Comment:
다른 객체에선 read-only 잘해주셨는데 이 부분은 놓치셨네요 :)
```suggestion
        return Collections.unmodifiableMap(result);
```

Reply:
감사합니다! 수정했습니다!

Reply:
엇... 수정했습니다! 👍